CHANGELOG.md

```diff
@@ -7,8 +7,12 @@ Unitwise uses semantic versioning.
 
 ## Unreleased
 
-- Added support for newer Rubies (2.6+) & drops support for older ones (2.6 is EOL but will continue to support for now)
-- Support Psych 4. Note: unit data YAML files in the gem are loaded using `unsafe_load`
+## 2.3.0 - 2022-10-08
```

Comment:
```suggestion
## 2.3.0 - 2022-10-19
```

Not sure if it really matters but yesterday was `2022-10-18` not `2022-10-08` 😄 ... also now I guess can update to 19th